### DISCUSSION TITLE
ОШИБКА: Что то пошло не так. Хук на стороне МойСклад отключен в нарушении настройки. Попробуйте отключить и включить снова. Если не поможет - обратитесь в техподдержку.

### DISCUSSION THREAD

**Discussion by Iamsoncha** [wpcraft-ru/wooms#543] [Q&A]
У меня две копии сайта, одна из которых перенастроена.
то тесть я имею два разных по идее сайта, которые тянут товары и заказы из разных баз Мой склад под разными логинами.
На одном хук работает, на другом - нет.

**Comment by aiiddqd** [comment]
2 сайта на 1 склад - история сложная.

консультации по сложным историям тут https://wpcraft.ru/ask/
